common/src/tx/mod.rs

```diff
@@ -101,15 +102,21 @@ impl Body {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, RlpEncodable, RlpDecodable, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, RlpEncodable, RlpDecodable, Serialize, Deserialize, Getters)]
 #[rlp(trailing)]
 pub struct Tx {
+    #[getset(get = "pub")]
     nonce: u64,
+    #[getset(get = "pub")]
     from: Address,
+    #[getset(get = "pub")]
     // Use 0x0 for transactions intended to be processed by the network
     to: Address,
+    #[getset(skip)]
     body: Body,
+    #[getset(get = "pub")]
     signature: Signature,
+    #[getset(skip)]
     sequence_number: Option<u64>,
 }
```

Comment:
To simplify things, lets just put #[getset(get = "pub")] on top of Tx, so it applies to all fields.
```suggestion
#[derive(Debug, Clone, PartialEq, RlpEncodable, RlpDecodable, Serialize, Deserialize, Getters)]
#[rlp(trailing)]
#[getset(get = "pub")]
pub struct Tx {
    nonce: u64,
    from: Address,
    // Use 0x0 for transactions intended to be processed by the network
    to: Address,
    body: Body,
    signature: Signature,
    sequence_number: Option<u64>,
}
```

The code should be updated to accommodate for this change.

Same applies to everywhere we use Getters.
For Setters, individual fields should be picked and made public, for safety reasons

Reply:
Done 1ccfa54